### ISSUE TITLE
Crash at startup

### ISSUE THREAD

**Issue by aknight2015** [devnoname120/vhbb#110]
I'm using 0.92. I get:
"An error has occurred in the following application.
(C2-12828-1)"
I can't even get to the logs. It just keeps crashing. It's the only app that does that.

**Comment by darkelement1987** [comment]
I also get crashes on startup, but the project looks dead?

**Comment by darkelement1987** [comment]
Crash log
[psp2core-1628799472-0x0000332011-eboot.bin.zip](https://github.com/devnoname120/vhbb/files/6978339/psp2core-1628799472-0x0000332011-eboot.bin.zip)

**Comment by BlackMage2** [comment]
crashes with me too (error C2-12828-1)
and after deletion  ux0:data/vhbb/, only the browser opens with 'Failed to view the page. (C2-14921-7)'
saved core file:
[psp2core-1636751735-0x0001813bc7-eboot.bin.psp2dmp.zip](https://github.com/devnoname120/vhbb/files/7530476/psp2core-1636751735-0x0001813bc7-eboot.bin.psp2dmp.zip)

and debug log creation fails even with C2-12828-1

opens vhbb http://detectportal.firefox.com/success.txt in the browser?

works vhbb not offline?

**Comment by robsdedude** [comment]
It does not work offline, no. It relies on https://vitadb.rinnegatamante.it/#/. How should it be able to download homebrew anyway if it's offline?

**Comment by Jimmisama** [comment]
![2022-07-22-000236-892890](https://user-images.githubusercontent.com/30177832/180261346-efbfbf12-f67f-4cb3-9e19-4e43c5ac39ea.png)
Crashed too, yesterday it opens some firefox test page everytime I open VHBB, at the same time I could not update from autoplugin 2 too. Today it crashed.  Here's two log files zipped in one
[psp2core-1658419633-0x00011f2def-eboot.bin.zip](https://github.com/devnoname120/vhbb/files/9160976/psp2core-1658419633-0x00011f2def-eboot.bin.zip)

 I could not get to logs too.

**Comment by BlackMage2** [comment]
> It does not work offline, no. It relies on https://vitadb.rinnegatamante.it/#/. How should it be able to download homebrew anyway if it's offline?

that's true, but that doesn't matter since it won't start anyway. It would also be important to know what error C2-12828-1 actually is. it's an error of an corrupted database. but how does this come about?

i have tried already all of the "Error C2-12828-1 fixes" at wololo.net

**Comment by Cyberbulat** [comment]
i might clone this progect and fix the errors in the progect but my vita sdk is broken 😔

**Comment by BlackMage2** [comment]
Then fix your Vita SDK?

**Comment by robsdedude** [comment]
@Cyberbulat if you manage to fix your SDK: try compiling VHBB without optimizations. I remember playing around with the optimizations options a few years back and without optimizations VHBB ran much more stable. Some optimization flags led to dumps that mentioned something about invalid instructions. So I have the suspicion that the build chain does some invalid optimizations. Sadly, I was not able to narrow it down further.

**Comment by Cyberbulat** [comment]
> @Cyberbulat if you manage to fix your SDK: try compiling VHBB without optimizations. I remember playing around with the optimizations options a few years back and without optimizations VHBB ran much more stable. Some optimization flags led to dumps that mentioned something about invalid instructions. So I have the suspicion that the build chain does some invalid optimizations. Sadly, I was not able to narrow it down further.

Ahh right thanks, I will folk this repo and I try to compile with out optimizations and I'll see if it runs smoothly or not

**Comment by BlackMage2** [comment]
> > @Cyberbulat if you manage to fix your SDK: try compiling VHBB without optimizations. I remember playing around with the optimizations options a few years back and without optimizations VHBB ran much more stable. Some optimization flags led to dumps that mentioned something about invalid instructions. So I have the suspicion that the build chain does some invalid optimizations. Sadly, I was not able to narrow it down further.
> 
> Ahh right thanks, I will folk this repo and I try to compile with out optimizations and I'll see if it runs smoothly or not

that's good to hear. please let us know about progress

**Comment by Cyberbulat** [comment]
> > > @Cyberbulat if you manage to fix your SDK: try compiling VHBB without optimizations. I remember playing around with the optimizations options a few years back and without optimizations VHBB ran much more stable. Some optimization flags led to dumps that mentioned something about invalid instructions. So I have the suspicion that the build chain does some invalid optimizations. Sadly, I was not able to narrow it down further.
> > 
> > Ahh right thanks, I will folk this repo and I try to compile with out optimizations and I'll see if it runs smoothly or not
> 
> that's good to hear. please let us know about progress

Will do don't worry 😂
